samples/outlook/35-notifications/add-getall-remove.yaml

```
@@ -55,20 +55,41 @@ script:
         function addInsight() {
           // Adds an informational message with actions to the mail item.
           const id = $("#notificationId").val().toString();
-          const details = 
-            {
+
+          let details;
+          // Identifies whether the current mail item is in read or compose mode.
+          if (Office.context.mailbox.item.itemId != undefined) {
+            // Read mode.
```

If you accept Kwame's suggestion re: scoping down the if/else, then you can do something like:
```suggestion
          let commandIdFromManifest = "PG.HelpCommand.Read";
          if (Office.context.mailbox.item.itemId == undefined) {
            commandIdFromManifest = "PG.HelpCommand.Compose";
          }
```